index.html

```diff
@@ -8,19 +8,19 @@
 <body>
```

Comment:
**Negeer deze pull request volledig!**
Ik had de tech it easy opdracht 4 weken geleden gemaakt en novi heeft deze opdracht aangepast.
Dit is dus niet de correcte versie!
https://github.com/wil004/Tech-It-Easy-Novi/pull/1/commits/8d73d92e228544079c8a264d8159b50b93edb979

**Dit is de link van de pull-request!**